akvo/rest/views/project_editor.py

```diff
@@ -572,6 +573,67 @@ def project_editor(request, pk=None):
 
 
 @api_view(['POST'])
+@permission_classes((IsAuthenticated,))
+def project_editor_reorder_items(request, project_pk=None):
+    """API call to reorder results or indicators"""
+
+    errors = []
+
+    item_type = request.POST.get('item_type', False)
+    item_id = request.POST.get('item_id', False)
+    item_direction = request.POST.get('item_direction', False)
+
+    # print 'Type: %s | ID: %s | Direction: %s' % (item_type, item_id, item_direction)
+
+    if item_type == 'result':
+        item_selected = Result.objects.get(id=item_id)
+        item_list = Result.objects.filter(project_id=project_pk)
+    elif item_type == 'indicator':
+        item_selected = Indicator.objects.get(id=item_id)
+        item_list = Indicator.objects.filter(result_id=item_selected.result_id)
+    else:
+        errors += ['Invalid item type']
+
+    # assign order if it doesn't already exist
+    if not item_list[0].order:
```

Comment:
If ```item_type``` is not 'result' or 'indicator', then ```item_list``` is not instantiated here, which will give a ```NameError```.